correct link to occ search commands (#4118)
owncloud/docs: modules/admin_manual/pages/configuration/search/index.adoc

service elasticsearch restart
----

NOTE: Only Elasticsearch version 5.6 is fully supported by version 1.0.0 of the Full Text Search app. 

> Version 7 of Elasticsearch will be supported by version 2.0.0 of the app.

Reviewer: ```suggestion
Version 7 of Elasticsearch is supported by version 2.0.0 of the app.
```
The future is soon present - or 😅
